crates/sui-indexer-alt/src/handlers/sum_packages.rs

```diff
@@ -16,8 +16,6 @@ use crate::{
     schema::sum_packages,
 };
 
-const CHUNK_ROWS: usize = i16::MAX as usize / 5;
```

Comment:
i.e. here, we should have 

```rust
const CHUNK_ROWS: usize = i16::MAX as usize / StoredPackage::FIELD_COUNT
```

this takes advantage of the field count trait, without adding an implicit coupling between the sequential pipeline traits and each sequential pipeline commit impl. today with this change we are assuming that sequential pipelines all do updates and deletes, and deletes all only require one bind parameter, this may not be true:

- If a table has multiple keys, then the deletes may need more bind parameters.
- We have discussed the idea of sequential pipelines that act kind of like pruners, which means their main work is to delete.
- We have also discussed the idea of separating the work for the sequential part of consistent pipelines into explicit updates and inserts, in that case, updates may have different bounds compared to rows if they are not setting all fields.